src/commands/randtools.js

```diff
@@ -0,0 +1,3 @@
+module.exports.randElem = array => array[this.randInt(0, array.length - 1)]
```

Comment:
Does this actually work?

Reply:
And, more importantly, will it continue working if the function `randElem` is added to an object?

Reply:
It does work. If you're iffy on it, we could always change it to `array[Math.floor(Math.random() * array.length)]`, I just thought it'd be cool to use the function within the module itself.

Reply:
I mean, reusing `randInt` is a good idea, but it can also be invoked as `module.exports.randInt` instead of `this.randInt`. But if you've tested that this works, it's fine.

Reply:
Huh, actually that's the lint that added that `this`, I hadn't noticed it.